include/dlt/dlt_common.h

```diff
@@ -1194,8 +1194,17 @@ void dlt_print_with_attributes(bool state);
 /**
  * Initialize (external) logging facility
  * @param mode positive, 0 = log to stdout, 1 = log to syslog, 2 = log to file, 3 = log to stderr
+ * @param overwrite_flag 1 = overwrite LoggingFilename, 0 = append LoggingFilename
  */
-void dlt_log_init(int mode);
+void dlt_log_init_p1(int mode, int overwrite_flag);
```

Comment:
Hello dbalym,
Could you tell me the reason why we need to separate the primitive API dlt_log_init into 2 others?
I am not so clear about this change.
Thank you